src/main/java/it/pagopa/pn/service/desk/action/impl/ValidationOperationActionImpl.java

```diff
@@ -114,6 +83,39 @@ public void execute(String operationId) {
                 .block();
     }
 
+    private Mono<Void> checkValidationFlow(PnServiceDeskOperations operation, PnServiceDeskAddress addres) {
+        return getIuns(operation.getRecipientInternalId())
+                .collectList()
+                .doOnNext(responsePaperNotificationFailed -> {
```

Comment:
utilizziamo il flatMap invece che il doOnNext in questi casi:

.flatMap(responsePaperNotificationFailed -> {
log.debug("listOfIuns = {}, List of iuns retrivied", responsePaperNotificationFailed);
operation.setErrorReason(null);
log.debug("errorReason = {}, Setting to null errorReason into entityOperation", operation);
return updateOperationStatus(operation, OperationStatusEnum.VALIDATION).thenReturn(responsePaperNotificationFailed);
})

Reply:
fixato